package.json

```diff
@@ -91,7 +91,7 @@
     "@babel/preset-react": "^7.0.0",
     "@cerner/browserslist-config-terra": "^3.1.0",
     "@cerner/eslint-config-terra": "^4.0.0",
-    "@cerner/terra-cli": "^1.0.0",
+    "@cerner/terra-cli": "^1.6.0",
     "@cerner/terra-functional-testing": "file:./cerner-terra-functional-testing-0.0.0.tgz",
```

Comment:
I found an issue with themes in terra-functional-testing 1.0.0 so I haven't updated this to 1.0.0 yet. I'm currently investigating the issue.